src/main/java/core/basesyntax/service/impl/StorageParserImpl.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.StorageParser;
+import java.util.Map;
+
+public class StorageParserImpl implements StorageParser {
```

Comment:
What this class really do - creates the report. So let's name it ReportServiceImpl